src/Entity/Phone.php

```diff
@@ -144,6 +152,61 @@ public function __construct()
         $this->updatedAt = new DateTimeImmutable();
     }
 
+    /**
+     * hydrate the Phone entity.
+     *
+     * @param null|string $constructor
+     * @param null|string $name
+     * @param null|float  $priceEuro
+     * @param null|string $system
+     * @param null|string $userInterface
+     * @param null|string $processor
+     * @param null|string $ram
+     * @param null|string $capacity
+     * @param null|string $das
+     * @param null|string $batteryCapacity
+     * @param null|bool   $wirelessCharging
+     * @param null|string $weight
+     * @param Size        $size
+     * @param Screen      $screen
+     *
+     * @return self
+     */
+    public function hydrate(
+        ?string $constructor,
+        ?string $name,
+        ?float $priceEuro,
+        ?string $system,
+        ?string $userInterface,
+        ?string $processor,
+        ?string $ram,
+        ?string $capacity,
+        ?string $das,
+        ?string $batteryCapacity,
+        ?bool $wirelessCharging,
+        ?string $weight,
+        Size $size,
+        Screen $screen
+    ): self {
+        $this->setConstructor($constructor);
+        $this->setName($name);
+        $this->setPriceEuro($priceEuro);
+        $this->setSystem($system);
+        $this->setUserInterface($userInterface);
+        $this->setProcessor($processor);
+        $this->setRam($ram);
+        $this->setCapacity($capacity);
+        $this->setDas($das);
+        $this->setBatteryCapacity($batteryCapacity);
+        $this->setWirelessCharging($wirelessCharging);
+        $this->setWeight($weight);
+        $this->setUuid(Uuid::uuid4());
```

Comment:
Codacy found an issue: [Avoid using static access to class '\Ramsey\Uuid\Uuid' in method 'hydrate'.](https://app.codacy.com/gh/CarolineDirat/P7APIBileMo/pullRequest?prid=6304570)